tests/vmi_multus_test.go

```diff
@@ -39,6 +36,12 @@ import (
 	"kubevirt.io/kubevirt/tests"
 )
 
+const (
+	postUrl    = "/apis/k8s.cni.cncf.io/v1/namespaces/%s/network-attachment-definitions/%s"
+	ovsConfCRD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s"},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"type\": \"ovs\", \"bridge\": \"br1\", \"vlan\": 100 }"}}`
```

Comment:
These configs already exist in the cluster in the default namespace right? Is there a reason we shouldn't perform a Get on the configs in the default namespace and then Post them to the kubevirt-test-default namespace?

Reply:
From @rmohr commend
> We should come up with a solution which does not depend on the default namespace in any way.

Reply:
ah, alrighty then. Makes sense.